governance/polls/Ratification Poll - MIP102c2-SP35 - June 10, 2024.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP35) - June 10, 2024
+summary: This subproposal amends MIP107 to define a process for stablecoin recovery.
+discussion_link: https://forum.makerdao.com/t/mip102c2-sp35-mip-amendment-subproposal-formal-submission/24423
+parameters:
+    input_format:
+        type: single-choice
+        abstain: [0]
+    victory_conditions:
+        - {
+            type: 'and',
+            conditions: [
+                { type : plurality },
+                { type : comparison, comparator : '>=', value: 10000 }
+            ]
+        }
+        - {type : default, value : 2 }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-06-10T16:00:00
+end_date: 2024-06-24T16:00:00
+---
+# Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP35) - June 10, 2024
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for fourteen days beginning on Monday, June 10 at 16:00 UTC.
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP102c2-SP35 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This subproposal will amend MIP107 Section 8.1, to include a strategy for the specifications of recoveries.*
+
+Please review the links below to inform your position on this proposal before voting.
+
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/19d671c7ba584aa573390c33e75e4f11204ec732/MIP102/MIP102c2-Subproposals/MIP102c2-SP35.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP102c2SP35)
```

Comment:
Please fix the formatting of block quotes in this proposal on the MIPs Portal, this will require a change to the canonical link.

![image](https://github.com/makerdao/community/assets/139177554/e800f9b5-9b20-4777-8092-be1dcc9e6c3b)